registry.go

```diff
@@ -89,8 +89,7 @@ func main() {
 	if err := cmd.Start(); err != nil {
 		log.Fatalf("Could not start node: %s", err)
 	}
-	// TODO: Does defer even work here?
-	defer func() {
+	go func() {
 		if err := cmd.Wait(); err != nil {
```

Comment:
I agree, but I tried to leave the "functionality" alone. I thought about running the whole "start node" in a loop and restart it whenever it stops. But unsure how the maintainers want the functionallity to be and how nicely that plays with the proxy.